dev/com.ibm.ws.cdi.jee_fat/publish/servers/cdi40SessionPersistenceServer/bootstrap.properties

```diff
@@ -0,0 +1,2 @@
+bootstrap.include=../testports.properties
```

Comment:
Missing copyright here

Reply:
The FAT here doesn't have copyrights here, so I'm excluding it too.